react/src/components/FormItemWithUnlimited.tsx

```diff
@@ -1,64 +1,78 @@
 import Flex from './Flex';
-import { Form, FormInstance } from 'antd';
-import Checkbox, { CheckboxChangeEvent } from 'antd/es/checkbox';
-import React from 'react';
+import { Form, Checkbox } from 'antd';
+import { CheckboxChangeEvent } from 'antd/es/checkbox';
+import { NamePath } from 'antd/es/form/interface';
+import _ from 'lodash';
+import React, { cloneElement, useEffect, useState } from 'react';
 import { useTranslation } from 'react-i18next';
 
 interface FormItemWithUnlimitedProps {
```

Comment:
You can use PropsWithChildren instead of explicit `children` props.
```
interface FormItemWithUnlimitedProps extends PropsWithChildren{
  name: NamePath;
  unlimitedValue?: number | string;
  label?: string;
}
```